RecoTracker/CkfPattern/plugins/GroupedCkfTrajectoryBuilder.cc

```diff
@@ -848,11 +848,11 @@ GroupedCkfTrajectoryBuilder::rebuildSeedingRegion(const TrajectorySeed&seed,
     // (e.g. because no Tracker layers outside seeding region) 
     //
 
-    if ( it->measurements().size()<=startingTraj.measurements().size() ) {
-      rebuiltTrajectories.push_back(std::move(*it));
-      LogDebug("CkfPattern")<< "RebuildSeedingRegion skipped as in-out trajectory does not exceed seed size.";
-      continue;
-    }
+    //if ( it->measurements().size()<=startingTraj.measurements().size() ) {
+    //  rebuiltTrajectories.push_back(std::move(*it));
+    //  LogDebug("CkfPattern")<< "RebuildSeedingRegion skipped as in-out trajectory does not exceed seed size.";
+    //  continue;
+    //}
```

Comment:
shouldn't this and the other one be made configurable?
That's unless we never ever make it into this branch in current tracking.